### PR TITLE
Change some medium services to easy or hard and update some urls

### PR DESCRIPTION
**Pull request by c106208**
Changed the difficulty of some sites to easy or hard from medium as they would be better classified under them. Also updated a few outdated urls. (#231)

### CODE REVIEW

File: _data/sites.json [hunk 1]
```diff
@@ -5223,9 +5217,9 @@
 
     {
         "name": "RunKeeper",
-        "url": "http://runkeeper.com/I-Love-RunKeeper",
-        "difficulty": "medium",
-        "notes": "After clicking the button, you will have to enter your password, complete a capcha, click another button, AND a confirmation modal dialog.",
+        "url": "https://runkeeper.com/delete-account?confirm",
+        "difficulty": "easy",
+        "notes": "Enter your password, select a reason for deletion, complete the captcha, press the delete button and then confirm it in the dialog.",
         "notes_it": "Dopo aver cliccato il bottone dovrai: inserire la tua password, completare un capcha, clicare un altro bottone E un dialogo di conferma.",
```
Comment (Nightfirecat) [inline _data/sites.json]: Since the English notes have changed, the translations should be updated or removed.

**Comment by c106208** [comment]
Done

File: _data/sites.json [hunk 2]
```diff
@@ -1959,9 +1959,9 @@
 
     {
         "name": "Epic Games",
-        "url": "http://fortnitehelp.epicgames.com/customer/en/portal/articles/2937300-how-do-i-delete-my-epic-account-",
+        "url": "https://epicgames.helpshift.com/a/epic-accounts/?s=epic-account-support&f=how-can-i-disable-my-epic-account&l=en&contact=1",
         "difficulty": "hard",
-        "notes": "You must contact them <a href=\"http://fortnitehelp.epicgames.com/customer/portal/emails/new\">via their contact form</a>.",
+        "notes": "Select 'Delete Epic Games Account' from the dropdown in the contact page",
```
Comment (tupaschoal) [inline _data/sites.json]: This looks like easy, doesn't it?

File: _data/sites.json [hunk 3]
```diff
@@ -4736,7 +4730,7 @@
     {
         "name": "Player FM",
         "url": "https://player.fm/will-miss-you",
-        "difficulty": "medium",
```
Comment (tupaschoal) [inline _data/sites.json]: It does need a lot of clicks, I'm wondering if I'd keep it medium.

**Comment by tupaschoal** [comment]
You mean Epic took 10 days to delete?
I didn't test the player.fm one, but if is simple as you described, it's ok to move it to easy,

**Comment by c106208** [comment]
Yeah, support takes about 5 days to respond to your first message and then requires you to respond with a statement confirming that you want to do so, and then takes a while to respond after that. I checked and it was more like 7/8 days, but still.

**Comment by tupaschoal** [comment]
I see, maybe we need to clarify the text then to state that it isn't
immediate and that you have to wait for contact.

**Comment by c106208** [comment]
Got it

**Comment by tupaschoal** [comment]
I'm ok with it, any more thoughts @Nightfirecat?

**Comment by PyroSamurai** [comment]
@c106208 thanks for sticking with PR, @tupaschoal it looks good to me.

**Comment by Nightfirecat** [comment]
LGTM as well. Thanks for addressing all of those changes!

**Pull request merged**